packages/manager/apps/web/client/app/hosting/hosting.html

```diff
@@ -89,7 +89,7 @@ <h1 class="d-inline-block mb-0 word-break">
         <oui-header-tabs>
             <oui-header-tabs-item
                 data-ng-if="$ctrl.tabs"
-                data-ng-repeat="tab in $ctrl.tabs track by $index"
+                data-ng-repeat="tab in $ctrl.tabs"
```

Comment:
Maybe we can track by something else ? an Id ? the tab title ?

Reply:
since it's an array of strings, i think the answer should be "track by $id(tab)" which is the exact same thing as not providing a track by.